### PR TITLE
Add last-modified timestamp to compliance docs

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:

* Adding last-modified-at plugin to better track changes to compliance-related docs.
* Added page information section with links to document history as commits in GitHub

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/peterb/last-mod-at)

## Security Considerations

We're adding a new plugin and changing the way Federalist processes our site. Reviewer should consult with me if this is a thing we want to do.

### CODE REVIEW

**Review by bengerman13 — DISMISSED** [review]

**Review by bengerman13 — DISMISSED** [review]

**Comment by pburkholder** [comment]
@bengerman13 Thanks for the reviews -- I should have marked it draft. I _am_ done now.

File: _docs/ops/security-ir-checklist.md [hunk 1]
```diff
@@ -89,3 +89,11 @@ If a cloud.gov team member or automated scanning system detects unauthorized acc
...
 
 If we've leaked secrets in Github, _once the Incident Response team has told us to remove the data from Github_, follow [these instructions to remove the secret from Git history](https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository), and then file a [support ticket to have the files removed from their cache](https://support.github.com/contact).
+
+---
+
+### Page information
+
+* Last modified on: {% last_modified_at %}
+* [Recent document history](https://github.com/cloud-gov/cg-site/commits/master/{{ page.path }}) (since 2020-02-05)
+* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05
```
Comment (ccostino) [inline _docs/ops/security-ir-checklist.md]: Missing an ending `)` it looks like:

```suggestion
* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05)
```

File: _docs/ops/configuration-management.md [hunk 2]
```diff
@@ -131,3 +131,11 @@ You can request a review using GitHub's built-in tools, mention someone in the P
...
 
 Whenever possible, the proposed changes should be tested locally. Because of the nature of many of the cloud.gov repositories and deployment environments, local testing is not always possible or practical. Visual code review, however, is always required.
+
+---
+
+### Page information
+
+* Last modified on: {% last_modified_at %}
+* [Recent document history](https://github.com/cloud-gov/cg-site/commits/master/{{ page.path }}) (since 2020-02-05)
+* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05
```
Comment (ccostino) [inline _docs/ops/configuration-management.md]: Missing an ending ) it looks like:

```suggestion
* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05)
```

File: _docs/ops/service-disruption-guide.md [hunk 3]
```diff
@@ -107,3 +107,11 @@ They also have a baseline expectation that we have good answers to the following
...
 * If it happens again, how are you going to notice faster and respond better next time?
 * Was there anything suspicious from a security perspective, or do you have confidence that all of the cause-and-effect steps were caused by ordinary reasons instead of malicious actors?
+
+---
+
+### Page information
+
+* Last modified on: {% last_modified_at %}
+* [Recent document history](https://github.com/cloud-gov/cg-site/commits/master/{{ page.path }}) (since 2020-02-05)
+* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05
```
Comment (ccostino) [inline _docs/ops/service-disruption-guide.md]: Missing an ending ) it looks like:

```suggestion
* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05)
```

**Review by ccostino — COMMENTED** [review]
Thanks, @pburkholder!  This is really cool.  It looks like some of the docs have a missing `)` at the end of the list, so I made some suggested edits.

Also, are there any other pages it would make sense to add this on?  I started looking in the Compliance section of our docs and didn't see it there (e.g., ATO Process, Auditing Activity, etc.).

**Comment by apburnes** [comment]
Would it make sense to ad this just once in the [docs layout](https://github.com/cloud-gov/cg-site/blob/master/_layouts/docs.html) and we would have this functionality on all of our docs?  It should only effect the `[Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md` path.

**Comment by ccostino** [comment]
I'm a fan of @apburnes' suggestion as it'll also reduce confusion on why some pages have this and others don't.

**Comment by pburkholder** [comment]
I like that suggestion a lot but I’ve already put in more time on this than it merited . So I think we should merge what we have unless someone has time and energy to come up with a good scheme that applies to all the appropriate pages.

**Review by ccostino — APPROVED** [review]
Thanks, @pburkholder!

@apburnes, we can follow up with another PR to shift this to the underlying layout so all pages get the feature in the future.

**Comment by onelittlebecca** [comment]
can we do both? Merge what we have now and add it into the docs layout for later?

**Pull request merged**